src/casscf/mcscf_2step.h

```diff
@@ -138,7 +143,10 @@ class MCSCF_2STEP {
     /// Solve CI coefficients for the current orbitals
     std::unique_ptr<ActiveSpaceSolver>
     diagonalize_hamiltonian(std::shared_ptr<ActiveSpaceIntegrals> fci_ints, const int print,
-                            double& e_c);
+                            const bool do_dipole, double& e_c);
```

Comment:
what is the last argument here? `double& e_c`? If this is an energy why not just grab it back from the returned `ActiveSpaceSolver` object? This mechanism of getting variables updated is not really ideal, unless there is a performance issue, which should not be the case here.

Reply:
Yes, `e_c` is the averaged energy. I guess I did not want to repeat the line of computing averaged energy, but I have made the changes.